.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Describe your changes
+
+## Issue ticket number and link
+
+### Checklist before requesting a review
+#### General
+- [ ] I have performed a self-review of my code
+- [ ] The commit message follows our guidelines
+- [ ] I have put the PR link into the planner
+- [ ] The PR has a clear and descriptive title and a detailed description of the changes
+
+#### Code Quality
+- [ ] Code Style
+- [ ] Linting
+- [ ] Documentation
+- [ ] Comments
+- [ ] No Console Logs/Debuggers
```

Comment:
I would condense this to something like "I have run linting and there were no errors" ticking boxes is gonna become annoying